subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/InProcessGradleExecuter.java

```diff
@@ -874,6 +874,11 @@ public void assertHasCause(String message) {
             }
         }
 
+        @Override
+        public void assertHasFirstCause(String message) {
```

Comment:
What about checking for causes.size() > 1 separately?

Otherwise, you'll get a strange assertion about a cause that's just an empty string.

Reply:
I applied your suggestion to both sites by extracting a common superclass.